src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,59 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int randomValue = 100;
+    private static final String defaultColor = "WHITE";
+    private static final int defaultRadius = 10;
```

Comment:
Don't forget about rules for constants naming:
```suggestion
    private static final int RANDOM_VALUE = 100;
    private static final String DEFAULT_COLOR= "WHITE";
    private static final int DEFAULT_RADIUS = 10;
```